File: python_backend/ranges/specialMoves/enPassant/get_en_passant_alignment_key.py
from custom_except import *
from coordType import *

def get_en_passant_alignment_key(sqr1, sqr2):
    """ takes 2 (x,y) coordinates as input, and returns their rankfiles concatinated"""
    rf1 = to_rankfile(sqr1)
    rf2 = to_rankfile(sqr2)
    return rf1 + rf2

if __name__ == "__main__":
    pass #TODO: implement test

File: node.js/ranges/specialMoves/enPassant/getEnPassantAlignmentKey.js
const  {xyToRf}  = require("../../../coordType/crdCnvrt");


function getEnPassantAlignmentKey(sqr1, sqr2) {
    /* takes 2 (x,y) coordinates as input, &&  returns their rankfiles concatinated**/
    let rf1 = xyToRf(...sqr1)
    let rf2 = xyToRf(...sqr2)
    return rf1 + rf2
}

module.exports = getEnPassantAlignmentKey;

if (require.main === module) {
    console.log(getEnPassantAlignmentKey([2, 4], [3, 4]))
}